app/src/main/java/org/openobservatory/ooniprobe/model/OONITests.java

```diff
@@ -13,7 +14,7 @@
     public static final String NDT_TEST = "ndt_test";
     public static final String COLLECTOR_ADDRESS = "https://b.collector.ooni.io";
     public static final String MAX_RUNTIME = "90";
-    public static final String WC_BACKEND = "https://b.web-connectivity.th.ooni.io";
-    public static final String HIRL_BACKEND = "http://a.echo.th.ooni.io";
-    public static final String HHFM_BACKEND = "http://a.http.th.ooni.io";
+    public static final String NOTIFICATION_SERVER_DEV = "https://registry.proteus.test.ooni.io";
+    public static final String NOTIFICATION_SERVER_PROD = "https://registry.proteus.ooni.io";
+    public static final String NOTIFICATION_SERVER = NOTIFICATION_SERVER_DEV;
```

Comment:
FIXME: This should be set to production. @lorenzoPrimi I was thinking that perhaps we can add in the future some switch to use testing rather than production environment without having every time to remember to fix individual files.